additional-databases/neptune/README.md

```diff
@@ -13,25 +13,29 @@ Make sure the EC2 instance is in the same VPC group as your Neptune cluster. To
 Lastly, make sure you save the key-pair file (.pem) and note the directory for use in the next step.
 
 ### Part 2: Set up an SSH tunnel.
-This step can vary depending on if you are running Windows or Mac.
+This step can vary depending on if you are running <b>Windows</b> or <b>Mac</b>.
 
-<b>Windows</b>
+1. Modify your hosts file to map localhost to your Neptune endpoint. 
 
-First, modify your hosts file as an Administrator (C:\Windows\System32\drivers\etc\hosts) to map localhost to your Neptune endpoint:
+    <b>Windows: </b> Open the hosts file as an Administrator (C:\Windows\System32\drivers\etc\hosts)
 
-127.0.0.1   localhost   your-Neptune-endpoint-here
+    <b>MacOS: </b> Open Terminal and type in the command: `sudo nano /etc/hosts`
 
-Next, open Command Prompt as an Administrator and navigate to the directory where you saved the EC2 key-pair file.  Run the following command:
+    Then add the following line to the hosts file.
+    `127.0.0.1   localhost   your-Neptune-endpoint-here`
 
-`ssh -i keypairfilename.pem ec2-user@yourec2instanceendpoint -N -L 8182:yourneptuneendpoint:8182`
+2. Open Command Prompt as an Administrator (or Terminal for MacOS) and navigate to the directory where you saved the EC2 key-pair file.  Run the following command:
```

Comment:
This navigation should not be necessary for Mac so long as they know where the pem file is. For Windows, SSH needs to be run from /Users/kelvin etc. doesn't it?